extension/submitting-a-log.md

```diff
@@ -10,21 +10,23 @@ position: 30
 
 # Submitting a Log
 
-Logs captured by the Fiddler Jam extension can be submitted to the [Fiddler Jam portal]({%slug fj-portal%}), shared with portal users via email, or saved as a [JSON-formatted archive (**HAR**)](https://en.wikipedia.org/wiki/HAR_(file_format)).
+Logs recorded by the Fiddler Jam extension can be submitted to the [Fiddler Jam portal]({%slug fj-portal%}) and shared with portal users via email or a direct link.
 
 ## Submit via Link
 
 Collaboration is the main feature of Fiddler Jam. You can easily submit a log in a few steps:
 
 1. [Record a log]({%slug fj-recording-logs%}).
 
-2. From the [**Link Generation options**]({%slug fj-link-options%}), select the access level you would like to apply to the recorded logs.
+2. From the [**Sharing options**]({%slug fj-link-options%}), set the access level for the recorded log (public or shared with specific people) and optionally protect the log with a password.
 
-   The default drop-down selection is **Anyone with this link**. You could also create a password-protected link by using the **Anyone with link and password** option or share the recorded logs with specific people via email by using the **Share with specific people** option.
+   The default drop-down selection is **Share with link**, which will create a public link. You could protect the log with a password by enabling the **Password protection** switch. The password must be at least eight characters long, must contain at least one lowercase, one uppercase, and one number. Click **Get Link** to submit the recorded Fiddler JAM log and generate a sharable link. The link points to the cloud space in the Jam portal that contains the submitted log. This URL is accessible only for Fiddler Jam [portal users]({%slug fj-users%}#portal-users) with the appropriate access rights.
```

Comment:
Remove the password details here and link to the newly created Password Protection section.